src/ralph/models/xapi/video/statements.py

```diff
@@ -170,6 +173,26 @@ class VideoTerminated(BaseVideoStatement):
     context: VideoTerminatedContext
 
 
+class VideoDownloaded(BaseVideoStatement):
+    """Pydantic model for video downloaded statement.
+
+    Example: John downloaded (rather than accessed or opened) a video.
+
+    Attributes:
+        verb (dict): See DownloadedVerb.
+        context (dict): See VideoDownloadedContext.
+    """
+
+    __selector__ = selector(
+        object__definition__type="https://w3id.org/xapi/video/activity-type/video",
+        verb__id="http://id.tincanapi.com/verb/downloaded",
+    )
+
+    verb: DownloadedVerb = DownloadedVerb()
+    result: Optional[BaseXapiResult]
```

Comment:
```suggestion
```
`Result` is not necessary as we don't overload `BaseXapiResult` for this statement